plot.go

```diff
@@ -338,10 +351,35 @@ func (plot *Plot) calcBrailleLines() {
 			continue
 		}
 
-		previousHeight := int((line[0] / plot.maxVal) * float64(height-1))
-
+		lastValWasNaN := math.IsNaN(line[0])
+		previousHeight := 0
+		if !lastValWasNaN {
+			previousHeight = calcDataPointHeight(line[0], plot.maxVal, 0, height)
+		}
 		for j, val := range line[1:] {
-			lheight := int((val / plot.maxVal) * float64(height-1))
+			if math.IsNaN(val) {
```

Comment:
This code could probably be written a bit simpler, it works as is though.
Feel free to optimize.